src/main/resources/META-INF/plugin.xml

```diff
@@ -73,6 +73,12 @@
                 <override-text place="GitMacheteContextMenu"/>
             </action>
 
+            <action id="GitMachete.MergeParentIntoSelectedBranchAction"
```

Comment:
I'd rather keep this action next to Rebase in the context menu (as they're very closely related and achieve the same goal)